qiling/os/posix/const_mapping.py

```diff
@@ -154,6 +154,17 @@ def socket_domain_mapping(p: int, archtype: QL_ARCH, ostype: QL_OS) -> str:
     return _constant_mapping(p, socket_domain_map)
 
 
+def socket_tcp_option_mapping(t: int, archtype: QL_ARCH, ostype: QL_OS) -> str:
```

Comment:
Even if we want to keep it consistent with other similar methods, `ostype` is not used here any may be removed.

Reply:
I not sure whether the option has diff on macos, like other methods. I'll just remove it first.